### DISCUSSION TITLE
Multiple Publications?

### DISCUSSION THREAD

**Discussion by ericpyle** [bible-technology/scripture-burrito#284] [Archive]
In the DBL, each Bible entry can have multiple licensable publications (book lists) associated with it. How does SB fit with that? Should each recipe have its notion of which ingredients are a part of that license? If so, are named groupings of ingredients allowed to be specified in the SB (e.g. so a licenser can select which to license, NT, NT+OT), or should each named grouping list of ingredients be in their own SB? [I suppose theoretically licensers may want to create their own named listings, but if so, I haven't heard that feedback yet]

**Comment by FoolRunning** [comment]
I think the idea is that the information required to know what is licensed should be a DBL-specific file inside the uploaded SB. When a user requests a copy of a given DBL entry, then DBL would generate a new SB specific to that user (with the required license and ingredients). The generated SB would then point back to the original SB as its source in the [relationships section](https://docs.burrito.bible/en/v0.3.1/schema_docs/relationships.html).

**Comment by ericpyle** [comment]
@FoolRunning Thanks, Tim.  That makes sense as far as the recipes go, but it doesn't make sense according to our current licensing workflow.

The current licensing workflow is 

1. translators specify the publications (aka canons) where each has their own id and book list and ordering, and 
2. then sometime after the upload to DBL, a licenser chooses from those named publications (or all books) to license to a publisher. (It's possible that publishers may also be able to select from those publications when requesting access from the licenser)

So, if licensers are no longer going to have a list of publications to select from at upload time, that will change the current workflow, and so I feel like licensers may need to be involved in this decision (e.g. It could be that they'd prefer to have such publications or booklists defined at an orgianization level, not at the entry level)
